packages/matter.js/src/codec/BtpCodec.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ByteArray, Endian } from "../util/ByteArray.js";
+
+export interface BTPHandshakeRequest {
+    sourceBits: SourceBits,
+    managementOpcode?: string,
+    version: ByteArray,
+    attMtu: ByteArray,
+    clientWindowSize: number,
+}
+
+export interface BTPHandshakeResponse {
+    sourceBits: SourceBits,
+    managementOpcode?: string,
+    reserved: Reserved,
+    finalProtocolVersion: ProtocolVersion,
+    attMtu: ByteArray,
+    selectedWindowSize: number,
+}
+
+export const enum SourceBits {
```

Comment:
I would use the Bit names and not the "abbreviations" ...

Reply:
okay, will update that 👍🏻